### ISSUE TITLE
Unable to retrieve a token

### ISSUE THREAD

**Issue by qnm** [jetrockets/omniauth-instagram-graph#1]
Hi!

Thanks for creating this gem. I'm having an issue getting it up-and-running though.

On the callback phase I consistently see the following:

```
Started GET "/auth/instagram_graph" for 49.255.230.53 at 2020-03-05 04:05:01 +0000
(instagram_graph) Request phase initiated.
Started GET "/auth/instagram_graph/callback?code=reg&state=erg" for 49.255.230.53 at 2020-03-05 04:05:05 +0000
(instagram_graph) Callback phase initiated.
(instagram_graph) Authentication failure! invalid_credentials: OAuth2::Error, :
{"error_type": "OAuthException", "code": 400, "error_message": "This authorization code has been used"}
Started GET "/auth/failure?message=invalid_credentials&strategy=instagram_graph" for 49.255.230.53 at 2020-03-05 04:05:08 +0000
Processing by SessionsController#failure as HTML
Parameters: {"message"=>"invalid_credentials", "strategy"=>"instagram_graph"}
```

Any ideas what I've missed?

**Comment by qnm** [comment]
This seems to have been fixed in https://github.com/CatchRelease/omniauth-instagram-graph/commit/9fb9aef85a1b27b4df1b5190af00e6444898004f

I think it's worth pulling that change in here!

**Comment by igor-alexandrov** [comment]
@euanlau can you please create a PR?

**Comment by henrikbjorn** [comment]
@igor-alexandrov You could also just pull the commit into manually. Would keep the attribution for CatchRelease.

**Comment by igor-alexandrov** [comment]
Fixed in #2. Release 1.0.2.

**Comment by igor-alexandrov** [comment]
Thanks!
